gravitee-apim-console-webui/src/management/api/api-traffic-v4/analytics/components/api-analytics-proxy-filter-bar/api-analytics-proxy-filter-bar.component.html

```diff
@@ -16,7 +16,9 @@
 
 -->
 <form [formGroup]="form" class="form">
+  <gio-select-search formControlName="httpStatuses" [options]="httpStatuses" label="HTTP Status"> </gio-select-search>
```

Comment:
Non-blocking comment: our Angular version allows auto-closing tags when there is no inner content specified 👍 
```suggestion
  <gio-select-search formControlName="httpStatuses" [options]="httpStatuses" label="HTTP Status" />
```